include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -1306,6 +1306,52 @@ FUNCTION(AutoDiffPopFromTape, swift_autodiffPopFromTape, C_CC,
          ARGS(OpaquePtrTy),
          ATTRS(NoUnwind))
 
+// SWIFT_ENABLE_TENSORFLOW
+FUNCTION(TF_NewStatus, swift_tfc_TF_NewStatus, C_CC,
+         RETURNS(Int8PtrTy),
+         ARGS(),
+         ATTRS(NoUnwind))
+
+FUNCTION(TF_DeleteStatus, swift_tfc_TF_DeleteStatus, C_CC,
+         RETURNS(),
+         ARGS(Int8PtrTy),
+         ATTRS(NoUnwind))
+
+FUNCTION(TFE_NewOp, swift_tfc_TFE_NewOp, C_CC,
+         RETURNS(Int8PtrTy),
+         ARGS(Int8PtrTy, Int8PtrTy, Int8PtrTy),
+         ATTRS(NoUnwind))
+
+FUNCTION(TFE_DeleteOp, swift_tfc_TFE_DeleteOp, C_CC,
+         RETURNS(),
+         ARGS(Int8PtrTy),
+         ATTRS(NoUnwind))
+
+FUNCTION(TFE_OpSetAttrType, swift_tfc_TFE_OpSetAttrType, C_CC,
+         RETURNS(),
+         ARGS(Int8PtrTy, Int8PtrTy, Int32Ty),
+         ATTRS(NoUnwind))
+
+FUNCTION(TFE_OpSetAttrTensor, swift_tfc_TFE_OpSetAttrTensor, C_CC,
+         RETURNS(),
+         ARGS(Int8PtrTy, Int8PtrTy, Int8PtrTy, Int8PtrTy),
+         ATTRS(NoUnwind))
+
+FUNCTION(CreateScalarFloatTensor, swift_tfc_CreateScalarFloatTensor, C_CC,
+         RETURNS(Int8PtrTy),
+         ARGS(Int32Ty),
+         ATTRS(NoUnwind))
+
+FUNCTION(TF_DeleteTensor, swift_tfc_TF_DeleteTensor, C_CC,
```

Comment:
Out of curiosity, why do you need these wrappers?  Why can't you just declare TF_DeleteTensor here with the `void*` flavored signature?  It should all work out due to the behavior of getRuntimeFunction calling LLVM's getOrCreateFunction API that returns the function or does a bitcast if an incompatible signature is already present.

Reply:
Things like _swift_tfc_CheckOk that are not trivial wrappers definitely will require entries here, I'm just asking about the trivial wrappers like the TF_DeleteTensor case.

Reply:
I'll try this and report back.

Reply:
Yes I'm able to remove the decls and impls in TensorFlow.{h,cpp}. The entries in RuntimeFunctions.def remain, so that we can call them in IRGen (e.g. via `IGM.getTF_DeleteTensorFn()`). Thanks!